coops-ui/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "coops-ui",
-  "version": "2.4.0",
```

Comment:
The 'deleted' side of the diff should show old version "2.5.0" but I think this is because you changed the target branch part way through. Anyway, I *think* this merge will still work correctly.

Reply:
This conversation should have been a comment instead of a review.

Lekshmi, feel free to resolve this conversation after you've read it. I will re-approve.

Reply:
Oh, this thing I noticed is why you're getting the merge conflict below. Sigh...

There's probably an easy way to fix this in GH, but if you can't find it then you'll need to rebase your source branch and re-apply the commits. If you want to wait until Monday then I can help you with branches.

Reply:
Fixed that issue by rebasing. Looks like just changing the target branch in the PR dint work as expected.

Reply:
Looks perfect now.